### DISCUSSION TITLE
Quick Question

### DISCUSSION THREAD

**Discussion by sneccc** [Tyrrrz/DiscordChatExporter#1258] [Q&A]
Hello while exporting images only,  i get avatars and gifs that are no desirable, is there a way to filter .gif or images below a certain height and width ? or even extract only the image links into a .txt to download them later would be even faster

**Comment by Tyrrrz** [comment]
It downloads all media assets referenced by the export. You can only filter out certain assets by filtering out the corresponding messages.
